lib/fluent/plugin/in_s3.rb

```diff
@@ -2,6 +2,7 @@
 require 'fluent/log-ext'
 
 require 'aws-sdk-resources'
+require 'cgi'
```

Comment:
Use `require 'cgi/util'`, please.
Becasue `require 'cgi'` loads all libraries related to CGI, but we don't use all cgi feature.